### DISCUSSION TITLE
Welcome to CSC110-Final-Project Discussions!

### DISCUSSION THREAD

**Discussion by scottctd** [UofT-CS-Group/CSC110-Final-Project#2] [Announcements]
Discussions are now open.

If we got any ideas, please put them here.

WeChat messages disappear very fast.

**Comment by CharlotteChenyy** [comment]
https://user-images.githubusercontent.com/60980117/140504324-6dadfe07-157b-4439-9815-2a6e8badbde8.mp4

1. 我把咱们图要的东西整好了 就是假如这两个点靠的特别近的话 需要先zoom in才能查看两个不同点的坐标
2. 我还有个问题是 咱们的散点会出现两个相同x对应两个不同y的情况吗? 如果可能的话 我还得改 因为我现在是按x坐标searchsorted的 所以相同的x值对应的y值只能识别到一个
